app/Providers/UpdateServiceProvider.php

```diff
@@ -45,30 +45,15 @@ protected function registerRoutes()
      */
     protected function registerConfig()
     {
-        $this->mergeConfigFrom(__DIR__.'/../Config/config.php', 'updater');
+        $this->mergeConfigFrom(__DIR__.'/../../config/updater.php', 'updater');
     }
 
     /**
      * Register views.
      */
     public function registerViews()
     {
-        $viewPath = resource_path('views/modules/updater');
-        $sourcePath = __DIR__.'/../Resources/views';
-
-        $this->publishes([
-            $sourcePath => $viewPath,
-        ], 'views');
-
-        $paths = array_map(
-            function ($path) {
-                return $path.'/modules/updater';
-            },
-            \Config::get('view.paths')
-        );
-
-        $paths[] = $sourcePath;
-        $this->loadViewsFrom($paths, 'updater');
+        $this->loadViewsFrom(__DIR__.'/views/updater', 'updater');
```

Comment:
Might not need to do this. I would move the views to being under `resources/views/system/` - so `updater`, `installer` and `importer` folders under there